### PR TITLE
Add Windows Setup in README.md

### PR DESCRIPTION
**Pull request by H4R5H1T-007**
Adding Support for windows virtual environment creation.

### CODE REVIEW

File: model_deployment/README.md [hunk 1]
```diff
@@ -19,15 +19,15 @@
...
     virtualenv venv
-    C:\> venv\Scripts\activate.bat
+    venv\Scripts\activate.bat
  ```
  
  #### 4.3 Setup virtual environment in Windows in PowerShell
   - Run the following commands in PowerShell.exe
  ```
     pip3 install virtualenv 
     virtualenv venv
```
Comment (pra17dod) [inline model_deployment/README.md]: This command should be written as `python3 -m virtualenv venv` in the both the methods of windows

File: model_deployment/README.md [hunk 2]
```diff
@@ -19,15 +19,15 @@
...
-    PS C:\> venv\Scripts\Activate.ps1
+    venv\Scripts\Activate.ps1
```
Comment (pra17dod) [inline model_deployment/README.md:33]: By default, external script execution is disabled in Powershell. Add commands to set execution policy.
`Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser`. For more details refer [here](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_execution_policies?view=powershell-7.1).

File: model_deployment/README.md [hunk 3]
```diff
@@ -19,15 +19,15 @@
  ```
     pip3 install virtualenv 
     virtualenv venv
-    C:\> venv\Scripts\activate.bat
+    venv\Scripts\activate.bat
  ```
```
Comment (pra17dod) [inline model_deployment/README.md:24]: Please add command to deactivate the virtualenv in all three methods. The command for that is `deactivate`

**Review by pra17dod — CHANGES_REQUESTED** [review]
Please do the necessary required to resolve the reviews. Try to squash all your commits into one to make the review process easier next time onwards.

File: model_deployment/README.md (hunk 3 above)
Comment (H4R5H1T-007) [reply model_deployment/README.md:24]: @pra17dod Is it necessary?

File: model_deployment/README.md [hunk 4]
```diff
@@ -17,18 +17,22 @@
...
+    pip3 -m install virtualenv 
     virtualenv venv
     venv\Scripts\activate.bat
  ```
  
  #### 4.3 Setup virtual environment in Windows in PowerShell
   - Run the following commands in PowerShell.exe
  ```
-    pip3 install virtualenv 
+    pip3 -m install virtualenv
```
Comment (pra17dod) [inline model_deployment/README.md]: There is no such command. It should be like this `python3 -m pip3 install virtualenv`.

File: model_deployment/README.md (hunk 1 above)
Comment (pra17dod) [reply model_deployment/README.md]: Please read my comment again and make changes accordingly.

**Review by pra17dod — CHANGES_REQUESTED** [review]

**Review by pra17dod — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**